src/main/java/core/basesyntax/PasswordValidationException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class PasswordValidationException extends RuntimeException {
```

Comment:
What is the difference between RuntimeException and Exception?

Reply:
Runtime - unchecked.
Exception - checked.
Wasn't attentive enough with Readme file.

Reply:
done